packages/dashboard-core-plugins/src/panels/NotebookPanel.tsx

```diff
@@ -511,6 +503,16 @@ class NotebookPanel extends Component<NotebookPanelProps, NotebookPanelState> {
     }
   }
 
+  getSettings = memoize(
+    (
+      initialSettings: editor.IStandaloneEditorConstructionOptions,
+      minimap: editor.IStandaloneEditorConstructionOptions
+    ) => ({
+      ...initialSettings,
+      ...minimap,
+    })
+  );
+
```

Comment:
Should just be `isMinimapEnabled` here, take a boolean. Taking a second `IStandaloneEditorConstructionOptions` doesn't make sense. Should also specify the return type as `IStandaloneEditorConstructionOptions`:
```suggestion
  getSettings = memoize(
    (
      initialSettings: editor.IStandaloneEditorConstructionOptions,
      isMinimapEnabled: boolean
    ): editor.IStandaloneEditorConstructionOptions => ({
      ...initialSettings,
      minimap: { enabled: isMinimapEnabled },
    })
  );
```